src/style.css

```
@@ -0,0 +1,93 @@
+@font-face {
+  font-family: "Avenir";
+  src: url(./fonts/Avenir-Book.ttf);
+  font-style: normal;
+  font-weight: 300;
+}
+
+@font-face {
+  font-family: "Avenir";
+  src: url(./fonts/Avenir-Heavy.ttf);
+  font-style: normal;
+  font-weight: 800;
+}
+
+.search-small,
+.search-big {
+  position: relative;
+  display: block;
+}
+
+.search-big {
+  margin-top: 20px;
+  margin-bottom: 20px;
+}
+
+.search-big input,
+.search-small input {
+  font-family: "Avenir", sans-serif;
```

apply font-family to `html` selector, and here just inherit styles
```suggestion
  font-family: inherit;
```